bpf-gpl/tc.c

```diff
@@ -1272,6 +1253,82 @@ static CALI_BPF_INLINE struct fwd calico_tc_skb_accepted(struct __sk_buff *skb,
 	}
 }
 
+static CALI_BPF_INLINE struct fwd calico_tc_skb_icmp_accepted(struct __sk_buff *skb, struct iphdr *ip, 
+						uint8_t type, uint8_t code)
+{
+	uint32_t fib_flags = 0;
+	__be32 un = 0;
+	int rc = TC_ACT_UNSPEC;
+	enum calico_reason reason = CALI_REASON_UNKNOWN;
+	uint32_t seen_mark;
+
+	if (code == ICMP_FRAG_NEEDED)
```

Comment:
we may just remove the other callers of `icmp_v4_reply` as they are not needed anymore, right?